frontend/src/App.jsx

```
@@ -1,43 +1,128 @@
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from "react-router-dom";
+import Home from "./pages/Home";
+import MyEvents from "./pages/MyEvents";
+import Events from './pages/Events';
+import Footer from './components/Footer.jsx';
+import { AuthProvider } from './context/AuthContext';
 import Navbar from './components/Navbar.jsx';
 import Footer from './components/Footer.jsx';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Footer'. Consider to delete it or rename it.

'Footer' is defined here:


(lint/suspicious/noRedeclare)